control-plane/.gitignore

```diff
@@ -0,0 +1,8 @@
+node_modules
+package-lock.json
+firebase-debug.log
+.firebase
+.github
+dist/404.html
+.firebaserc
+yarn.lock
```

Comment:
This one should be tracked by git as well, similar to `package-lock.json`
https://stackoverflow.com/questions/39990017/should-i-commit-the-yarn-lock-file-and-what-is-it-for